src/jrd/extds/InternalDS.cpp

```diff
@@ -190,7 +190,7 @@ void InternalConnection::attach(thread_db* tdbb)
 
 	memset(m_features, false, sizeof(m_features));
 	static const info_features features[] = ENGINE_FEATURES;
-	for (int i = 0; i < sizeof(features); i++)
+	for (int i = 0; i < sizeof(features)/sizeof(info_features); i++)
 		setFeature(features[i]);
```

Comment:
We have FB_NELEM macro exactly for this purpose.

Reply:
Oops, I forgot about it. Thanks.